include/net/nrf_cloud.h

```diff
@@ -316,6 +330,18 @@ int nrf_cloud_disconnect(void);
  */
 void nrf_cloud_process(void);
 
+/**
+ * @brief The application has handled re-init after a modem FOTA update and the
+ *        LTE link has been re-established.
+ *        This function must be called in order to complete the modem update.
+ *
+ * @param[in] fota_success true if modem update was successful, false otherwise.
+ *
+ * @retval 0 If successful.
+ *           Otherwise, a (negative) error code is returned.
+ */
+int nrf_cloud_modem_fota_completed(const bool fota_success);
```

Comment:
Does libmodem need to be reinitialized as well?  Or maybe that's done in dfu_target_modem?

Reply:
yes, `nrf_modem_lib_shutdown` and `nrf_modem_lib_init` need to be called.  how this is done is up to the application.